Orange/statistics/util.py

```diff
@@ -353,7 +354,10 @@ def weighted_mean():
 
 def _nan_min_max(x, func, axis=0):
     if not sp.issparse(x):
-        return func(x, axis=axis)
+        with warnings.catch_warnings():
```

Comment:
This currently still changes the behaviour of nanmin/nanmax compared to numpy's version.
I thought we agreed that for numpy arrays our functions should be equivalent to np.nanmin/max